src/components/Theme.tsx

```diff
@@ -5,6 +5,25 @@ import {
   experimental_extendTheme as extendTheme,
 } from "@mui/material/styles";
 
+declare module "@mui/material/styles" {
+  interface PaletteColor {
+    icon?: string;
+  }
+
+  interface SimplePaletteColorOptions {
+    icon?: string;
+  }
+}
+
+declare module "@mui/material/Typography" {
+  interface TypographyPropsVariantOverrides {
+    title: true;
+    subtitle: true;
+    error: true;
+    link: true;
+  }
+}
+
 const theme = extendTheme({
   typography: {
     fontFamily: "'IBM Plex Sans', sans-serif",
```

Comment:
Do you need to include the font somewhere? Won't this just default back to sans-serif?

Reply:
The font is declared in the theme object on line 29 :)

Reply:
Yes but if the theme isn't included from Google fonts or fontsource or wherever, it will only work if the user has the font installed on their machine otherwise it defaults back to 'san-serif'.

Reply:
Ooooh sorry I'm with you now 😅 The font is downloaded in a `link` tag in [index.html](../tree/production/public/index.html) :)